src/main/java/com/solvd/webtesting/page/HomePage.java

```diff
@@ -0,0 +1,90 @@
+package com.solvd.webtesting.page;
+
+import com.qaprosoft.carina.core.foundation.webdriver.decorator.ExtendedWebElement;
+import com.qaprosoft.carina.core.gui.AbstractPage;
+import com.solvd.webtesting.elements.ProductMenuBar;
+import com.solvd.webtesting.elements.SearchSection;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+public class HomePage extends AbstractPage {
+
+    @FindBy(xpath = "//*[contains(@class,'styles_primaryButton__30pEq')]")
+    private ExtendedWebElement cookieButton;
+
+    @FindBy(xpath = "//*[contains(@class,'styles_userToolsToggler__imcSl')]")
+    private ExtendedWebElement accountMenu;
+
+    @FindBy(css = ".userToolsBtn")
+    private ExtendedWebElement loginButton;
+
+    @FindBy(css = ".styles_userTools__2J7cp.undefined")
+    private ExtendedWebElement accountMenuAfterLogin;
+
+    @FindBy(css = ".userToolsSubtitle")
+    private ExtendedWebElement userToolsTitle;
+
+    @FindBy(xpath =  "//*[contains(@class,'styles_headerReactWrapper__3U15a') and contains(@class,'styles_headerReactWrapperAlt__3bM5y')]")
+    private com.solvd.webtesting.elements.ProductMenuBar ProductMenuBar;
+
+    @FindBy(xpath = "//*[contains(@class,'styles_headerReactLine__IWu_8') and contains(@class,'styles_headerReactLineSecond__tp1U0')]")
+    private com.solvd.webtesting.elements.SearchSection SearchSection;
+
+    public HomePage(WebDriver driver) {
+        super(driver);
+        setPageURL("");
+    }
+
+    public void clickCookieButton() {
+        cookieButton.click();
+    }
+
+    public void clickAccountMenu() {
+        accountMenu.click();
+    }
+
+    public LoginPage clickLoginButton() {
+        if (loginButton.isElementPresent() & loginButton.isClickable()) {
+            loginButton.click();
+        }
+        return new LoginPage(getDriver());
+    }
+
+    public void clickAccountMenuAfterLogin() {
+        if (accountMenuAfterLogin.isElementPresent() & accountMenuAfterLogin.isClickable()) {
+            accountMenuAfterLogin.click();
+        }
+    }
+
+    public String getAccountInfo() {
+        return userToolsTitle.getText();
+    }
+
+    public ExtendedWebElement getCookieButton() {
+        return cookieButton;
+    }
+
+    public ExtendedWebElement getAccountMenu() {
+        return accountMenu;
+    }
+
+    public ExtendedWebElement getLoginButton() {
+        return loginButton;
+    }
+
+    public ExtendedWebElement getAccountMenuAfterLogin() {
+        return accountMenuAfterLogin;
+    }
+
+    public ExtendedWebElement getUserToolsTitle() {
+        return userToolsTitle;
+    }
+
+    public ProductMenuBar getProductMenuBar() {
+        return ProductMenuBar;
+    }
```

Comment:
The same: Please don't return ExtendedWebElement, if use need to do some manipulations with elements, u need to do this on page where it present.